### PR TITLE
chore: A taste of linting (ose.js, ose.scss.js, == to ===)

### PR DESCRIPTION
**Pull request by wyrmisis**
## Intent

This changeset is meant to show what kinds of changes we can expect to see as we work through linting:
* Changing `==` to `===` and `!=` to `!==`
* Adding some JSDoc frontmatter for file summaries
* Sorting imports
* Changing `let` to `const` (when reasonable)
* Changing from loops to array iteration (where reasonable)
* Avoiding mutation of arguments (where possible)

### CODE REVIEW

File: src/ose.js [hunk 1]
```diff
@@ -1,40 +1,46 @@
...
+import OseDataModelWeapon from './module/item/data-model-weapon';
+import { OseItem } from "./module/item/entity";
+import { OseItemSheet } from "./module/item/item-sheet";
+import * as macros from "./module/macros";
+import * as party from "./module/party";
+import { OsePartySheet } from "./module/party/party-sheet";
+import { preloadHandlebarsTemplates } from "./module/preloadTemplates";
+import * as renderList from "./module/renderList";
+import { registerSettings } from "./module/settings";
+import * as treasure from "./module/treasure";
```
Comment (anthonyronda) [inline src/ose.js:33]: imports are alphabetical by filepath? Do we have other options?

Nothing particularly wrong with it from my side, just seems like it's not helpful. Any option for putting all directories first like so?
image

**Review by anthonyronda — APPROVED** [review]
I only have the one question, otherwise LGTM

File: src/ose.js (hunk 1 above)
Comment (wyrmisis) [reply src/ose.js:31]: I think I got something like this going.

File: src/ose.js [hunk 2]
```diff
@@ -114,27 +114,20 @@ Hooks.once("init", async () => {
...
 Hooks.once("setup", () => {
   // Localize CONFIG objects once up-front
-  [
-    "saves_short",
-    "saves_long",
-    "scores",
-    "armor",
-    "colors",
-    "tags",
-  ].forEach((o) => {
```
[the remark this answers is not in the document]
Comment (anthonyronda) [reply src/ose.js]: (not worth fixing this time)

**Pull request merged**